extensions/escape-game/standalone.mjs

```diff
@@ -510,6 +506,7 @@ class Standalone {
    */
   async s2s_sf_showTerminal(srcExt, playerID, kwargs, sfInfo) {
     await this._finalizeIfNeeded(playerID);
+    await this.playerToTeam.get(playerID).ensureContainerAvailable(sfInfo.visibleName);
```

Comment:
Consider using `sfInfo.name` instead of `sfInfo.visibleName`, the later is only for display.